### ISSUE TITLE
Buttons count is not adjusting based on the length of the button text.

### ISSUE THREAD

**Issue by rsjonnalagadda** [Gavras/MultiLineRadioGroup#9]
If the length of the text is too long and the max_in_row is set to 2, then I expect the buttons in row is 1.

**Comment by Gavras** [comment]
You are right, this library only alligns buttons based on the user exact request and does not calculate acual size of buttons.
Maybe in future release it will be included.
Thanks for your comment
